tests/django/manage.py

```diff
@@ -29,4 +29,7 @@
 
     from django.core.management import execute_from_command_line
 
+    os.environ.setdefault(u'HYPOTHESIS_DJANGO_USETZ_TEST', u'TRUE')
```

Comment:
Good old print-debugging confirms that you're right to be suspicious.  I don't know anything about Django tests though, and can't run stuff with Tox locally.  Any ideas?